custom-pages/process-autogenerated-form/src/js/app.js

```diff
@@ -11,9 +11,10 @@
   app.controller('MainCtrl', ['$scope', '$location', 'contractSrvc', '$window', 'processAPI', 'gettextCatalog', function ($scope, $location, contractSrvc, $window, processAPI, gettextCatalog) {
 
     var processId = $location.search().id;
-    $scope.inputArray = [];
+
     $scope.contract = {};
     $scope.dataToSend = {};
+    $scope.parent = $scope.dataToSend;
     $scope.autofill = {};
     $scope.process = {};
     $scope.message = undefined;
```

Comment:
should avoid polluting $scope and use controllerAs syntax with vm variable
[Y030](https://github.com/johnpapa/angular-styleguide#style-y030)
[Y031](https://github.com/johnpapa/angular-styleguide#style-y031)
[Y032](https://github.com/johnpapa/angular-styleguide#style-y032)